tapeagents/finetune/rl/__init__.py

```diff
@@ -197,6 +174,13 @@ def update_advantages(dataset: Dataset, config: RLConfig) -> Dataset:
     """
     df = dataset.to_pandas()
 
+    if config.implicit_kl_coef > 0:
```

Comment:
This adds negative KL to reward, I suppose. Can we call it `config.reward_minus_kl`?